backend/src/schema/resolvers/reports.spec.js

```diff
@@ -1,37 +1,73 @@
 import { GraphQLClient } from 'graphql-request'
 import Factory from '../../seed/factories'
-import { host, login } from '../../jest/helpers'
-import { neode } from '../../bootstrap/neo4j'
+import { host, login, gql } from '../../jest/helpers'
+import { getDriver, neode } from '../../bootstrap/neo4j'
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../.././server'
 
 const factory = Factory()
 const instance = neode()
+const driver = getDriver()
 
-describe('report', () => {
+describe('report mutation', () => {
   let reportMutation
   let headers
-  let returnedObject
+  let client
   let variables
   let createPostVariables
   let user
   const categoryIds = ['cat9']
 
+  const action = () => {
+    reportMutation = gql`
+      mutation($resourceId: ID!, $reasonCategory: ReasonCategory!, $reasonDescription: String!) {
+        report(
+          resourceId: $resourceId
+          reasonCategory: $reasonCategory
+          reasonDescription: $reasonDescription
+        ) {
+          createdAt
+          reasonCategory
+          reasonDescription
+          type
+          submitter {
+            email
+          }
+          user {
+            name
+          }
+          post {
+            title
+          }
+          comment {
+            content
+          }
+        }
+      }
+    `
+    client = new GraphQLClient(host, {
+      headers,
+    })
+    return client.request(reportMutation, variables)
+  }
+
   beforeEach(async () => {
-    returnedObject = '{ id }'
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T19:29:44Z" title="Monday, October 14th 2019, 9:29:44 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -480,7 +475,6 @@ describe('reports query', () => {
             submitter: expect.objectContaining({
               id: 'user1',
             }),
-            resourceId: 'auth1',
```


I was in a hurry and not concentrated … 😬

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T19:32:08Z" title="Monday, October 14th 2019, 9:32:08 pm +02:00">Oct 14, 2019</time>_

---


well, it works for queries, but didn't work for mutations, we could easily add it to the mutations as well, I just didn't feel like it's necessary, and didn't hear back from you so I just thought I'd make the change and revert if people wanted me to